core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3466,7 +3467,7 @@ payment.amazonGiftCard.info=To pay with Amazon eGift Card, you will need to send
   - try to use creative, believable text for the gift card''s message (e.g., "Happy birthday Susan!") along with the trade ID (and use trader chat \
   to tell your trading peer the reference text you picked so they can verify your payment)\n\
   - Amazon eGift Cards can only be redeemed on the Amazon website they were purchased on (e.g., a gift card purchased on amazon.it can only be redeemed on amazon.it)
-
+payment.capitual.info=Include any special instructions for the payment method that both buyer and seller might need to be aware of
```

Comment:
FYI this line actually shows in the software...it should probably be removed.

![Screenshot from 2021-06-15 23-48-32](https://user-images.githubusercontent.com/735155/122155243-921e1b80-ce34-11eb-8b20-e8f60cbd6106.png)

Reply:
So, no popup or different text?

Reply:
I would say if there's no text worth putting there, there should be no pop-up.

Reply:
Yes no special text needed.

Reply:
Please remove this unused translation.